stars/intel.py

```diff
@@ -47,6 +48,7 @@ def add_report(self, reference, date, report):
         # Special handling for locations to remove relative and reduce memory size
         if 'location' in report:
             report['location_root'] = report['location'].root_location.xyz
+            report['reference_root'] = Reference(reference=report['location'].root_reference)
```

Comment:
Why does intel need a reference to the reference_root?  The whole point of intel is to be a very static piece of info